packages/hardhat-plugin/src/index.ts

```diff
@@ -311,6 +338,90 @@ ignitionScope
     }
   );
 
+ignitionScope
+  .task("verify")
+  .addPositionalParam("deploymentId", "The id of the deployment to verify")
+  .setDescription(
+    "Verify contracts from a deployment against the configured block explorers"
+  )
+  .setAction(async ({ deploymentId }: { deploymentId: string }, hre) => {
+    const { verify } = await import("@nomicfoundation/ignition-core");
+    const { Etherscan } = await import(
+      "@nomicfoundation/hardhat-verify/etherscan"
```

Comment:
This should be imported into the top-level scope of this file. It will import its type-extensions, so we wouldn't need ours.

If you depend on another plugin, you should always do this, as you may be depending on one of its config validation/environment extensions/tasks/etc.

Reply:
I have pulled it in directly. Note this required me to bump the version of HH toolbox used in the samples.